### PR TITLE
Update Japanese translation

### PR DESCRIPTION
**Pull request by toshixm**
- merged up-to-date version of msg_hash_us.[ch] into msg_hash_ja.[ch].
- applied coding style using .editorconfig.
- added more translation messages.

### CODE REVIEW

**Comment by orbea** [comment]
C89_BUILD error.
```
ntl/msg_hash_ja.c:2150:16: error: string literal of length 536 exceeds maximum
      length 509 that C90 compilers are required to support
      [-Werror,-Woverlength-strings]
               "RetroArch自身だけでは何も実行できません。 \n"
               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [obj-unix/release/intl/msg_hash_ja.o] Error 1
```

@twinaphex How do we solve these?

**Comment by twinaphex** [comment]
You have to do the same trick that I did in other translation files - split them up into two separate char arrays of around length 509, then combine them later on.

See this for reference -

https://github.com/libretro/RetroArch/blob/master/intl/msg_hash_us.c#L704

**Comment by toshixm** [comment]
Thank you for advice.
I have fixed the problem and committed.

File: intl/msg_hash_ja.c [hunk 1]
```diff
@@ -625,41 +696,37 @@ int menu_hash_get_help_jp_enum(enum msg_hash_enums msg, char *s, size_t len)
          break;
       case MENU_ENUM_LABEL_VALUE_MENU_ENUM_CONTROLS_PROLOG:
          snprintf(s, len,
-               "You can use the following controls below \n"
-               "on either your gamepad or keyboard in order\n"
-               "to control the menu: \n"
-               " \n"
+            "メニューを操作するのに \n"
```
Comment (bparker06) [inline intl/msg_hash_ja.c]: I'm not an expert in Japanese, but is this supposed to be `には` instead of `のに` or does that work too?

Comment (toshixm) [reply intl/msg_hash_ja.c]: Thank you for your good suggestions.
I think the meaning is the same in either case, but will ask my friends about which one is better.
If necessary, I will fix it on the next PR of more translation messages.

**Pull request merged**